fcrepo4/fcrepo4
# Enables WebAC by default.

Note:  this commit currently breaks the "clickability" of the one click.
See modifications to the README for new instructions. FWIW I tried a number of different
approaches including ensuring that the the filtered jetty-one-click.xml  ended up in the
jetty console jar as  WEB-INF/jetty.xml.  To no avail.

Resolves  https://jira.duraspace.org/browse/FCREPO-2631

# What's new?
Basically this PR brings over webac the configuration from fcrepo-webapp-plus into the fcrepo-webapp module, ensuring that users who fire up fcrepo-webapp using the mvn jetty:run or the jetty console jar are able to see the webac framework in action.

# How should this be tested?
Fire up the webapp as you normally would.  See README for the jetty console launch command.
You can login using fedoraAdmin:fedoraAdmin as your credentials.

# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo4/committers
